snafu/upgrade_openshift_wrapper/trigger_upgrade.py

```diff
@@ -25,7 +25,6 @@ class Trigger_upgrade():
     def __init__(self, args):
         self.uuid = args.uuid
         self.user = args.user
-        self.version = args.version
```

Comment:
I think we still need this variable,  `self.version` is referenced in the line 92, to trigger an upgrade to a specific version